WEB_SUPPORT.md

```
@@ -0,0 +1,69 @@
+# Web Platform Support
+
+## Overview
+
+As of version 1.1.20, the `newrelic_mobile` package includes conditional exports that allow it to compile on Flutter Web (including WASM) without runtime errors. However, **New Relic Mobile monitoring is only functional on native mobile platforms (iOS and Android)**.
+
+## Behavior by Platform
+
+### Native Platforms (iOS/Android)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)